.circleci/config.yml

```diff
@@ -80,7 +81,9 @@ jobs:
 
               # on error, run this function
               exit_ssh() {
+                echo "inside trap exit_ssh"
                 echo $?
+                exit $?
```

Comment:
this was the other problem, we need to exit with this status code

Reply:
Here you're returning the `$?` of `echo $?` which should always be `0`.